maps/sccv_horizon/sccv_horizon-2_deck_2.dmm

```diff
@@ -25689,9 +25704,9 @@
 /turf/simulated/floor/tiled,
 /area/engineering/lobby)
 "mtP" = (
-/obj/machinery/camera/network/service{
-	c_tag = "Dinner - Kitchen Freezer";
-	dir = 4
+/obj/machinery/alarm{
```

Comment:
```suggestion
/obj/machinery/alarm/freezer{
```
Otherwise it won't cool the freezer properly.

Reply:
this should be mapped in, for the PR creator's note.